apps/site/src/lib/components/forms/shared/BaseForm.tsx

```diff
@@ -42,6 +47,9 @@ export default function BaseForm({ children }: PropsWithChildren) {
 			}
 		}
 
+		// attach application type to formData
+		formData.append("application_type", applicationType);
```

Comment:
Issue [marginal]: for any users with JavaScript disabled (should be very rare), the additional processing won't run, so this field will be missed. A more versatile solution is having a hidden input field in the form, see `GuestLoginVerificationForm` for an example.